src/shared/modules/discovery/discoveryDuck.test.ts

```diff
@@ -259,14 +259,14 @@ describe('discoveryOnStartupEpic', () => {
       url: 'http://localhost/?connectURL=bolt%2Brouting%3A%2F%2Fmyhost%3A8889'
     }
     const expectedURL = 'bolt+routing://myhost:8889'
-    bus.take(discovery.DONE, currentAction => {
+    bus.take(discovery.DONE, _currentAction => {
```

Comment:
Why the _ prefix? It seems used?

Reply:
Accidentally renamed instances in all test cases instead of just the ones were it was needed, reverting a few